pkg/model/translation_test.go

```diff
@@ -75,8 +77,8 @@ func TestDevToTranslationRule(t *testing.T) {
 
 	rule1 := dev.ToTranslationRule(dev, "n", "username", false)
 	rule1OK := &TranslationRule{
-		Marker:            OktetoBinImageTag,
-		OktetoBinImageTag: OktetoBinImageTag,
+		Marker:            config.NewImageConfig(io.NewIOController()).GetBinImage(),
```

Comment:
I'm not sure tests code and validation should relay on the actual function implementation instead of just the expected value. If there is a bug in the function logic, that is propagated to the unit tests